ntc_templates/templates/cisco_asa_show_running-config_object_network.textfsm

```diff
@@ -19,5 +19,5 @@ Start
   ^\s+subnet\s+${NETWORK}\/${PREFIX_LENGTH}\s*
   ^\s+range\s+${START_IP}\s+${END_IP}\s*
   ^\s+host\s+${HOST}\s*
-  ^\s+fqdn\s+${FQDN}\s*
+  ^\s+fqdn\s(?:v4|v6)?\s*${FQDN}\s*
```

Comment:
It doesn't hurt to keep the `\s+` between FQDN and the optional v4 or v6 just in case there's ever more than one whitespace character.

```suggestion
  ^\s+fqdn\s+(?:v4|v6)?\s*${FQDN}\s*
```